spec/system/community/community_spec.rb

```diff
@@ -108,6 +108,7 @@
     within("div#post-show-#{new_reply.id}") do
       replace_markdown reply_body_for_edit
     end
+    fill_in 'edit-reason', with: 'Test Edit Reason'
```

Comment:
As I'd mentioned in my original comment, let's **not** modify an existing scenario to test this feature.

While adding it to an existing scenario is the _easy_ thing to do, incremental changes like these make specs increasingly difficult to maintain over time, as the size of scenarios grow.

Also, independent scenarios allow us to test the full range of a feature. For example, the bug above wasn't caught because there are two sources of edit reasons, the `post` and `text_versions`. An independent scenario that edits a post twice and then checks the history page for both reason strings would check the feature's functionality properly...